workshops/forms.py

```diff
@@ -56,15 +57,43 @@ class InstructorsForm(forms.Form):
         lookup_class=lookups.AirportLookup,
         label='Airport',
         required=False,
-        help_text=AUTOCOMPLETE_HELP_TEXT,
+        widget=selectable.AutoComboboxSelectWidget(
+            lookup_class=lookups.AirportLookup,
+        )
```

Comment:
Please add a trailing comma in this line. To do that, start an interactive rebase, edit your commit which changes this line (bdc7aae) and then amend that commit.

More info here: https://www.youtube.com/watch?v=4EOZvow1mk4